setup.py

```diff
@@ -43,6 +43,7 @@
         'django>=3.2,<4',
         'boto3>=1.7',
         'cobalt>=6.1',
```

Comment:
should we remove this from indigo, given bluebell has cobalt 7 and higher in its dependencies?

Reply:
nice, done.